src/styles/days_settings.scss

```diff
@@ -0,0 +1,56 @@
+.calendar--start-day-mon {
+  .calendar__day:nth-child(1) {
+    margin-left: 0.5px;
+  }
+}
+
+.calendar--start-day-tue {
+  .calendar__day:nth-child(1) {
+    margin-left: 101.5px;
+  }
+}
+
+.calendar--start-day-wed {
+  .calendar__day:nth-child(1) {
+    margin-left: 202.5px;
+  }
+}
+
+.calendar--start-day-thu {
+  .calendar__day:nth-child(1) {
+    margin-left: 303.5px;
+  }
+}
+
+.calendar--start-day-fri {
+  .calendar__day:nth-child(1) {
+    margin-left: 404.5px;
+  }
+}
+
+.calendar--start-day-sat {
+  .calendar__day:nth-child(1) {
+    margin-left: 505.5px;
+  }
+}
+
+.calendar--start-day-sun {
+  .calendar__day:nth-child(1) {
+    margin-left: 606.5px;
+  }
+}
```

Comment:
You repeat the same CSS for every day of the week, making the code difficult to understand, manage, and maintain.
The code becomes longer and more complex.

Better use a Sass (CSS preprocessor) approach to generate styles for the calendar depending on its various properties.